prime-angular-frontend/src/app/modules/paper-enrolment/pages/obo-sites-page/obo-sites-form.model.ts

```diff
@@ -1,3 +1,9 @@
-import { HttpEnrollee } from '@shared/models/enrolment.model';
+import { OboSite } from '@enrolment/shared/models/obo-site.model';
+import { CareSetting } from '@enrolment/shared/models/care-setting.model';
+import { EnrolleeHealthAuthority } from '@shared/models/enrollee-health-authority.model';
 
-export interface OboSitesFormModel extends Pick<HttpEnrollee, 'oboSites'> { }
+export interface OboSitesFormModel {
```

Comment:
Looks like you removed:

```
export interface OboSitesFormModel extends Pick<HttpEnrollee, 'oboSites'> { }
```

These keys are all on `HttpEnrollee` so you can add them all to the `Pick` instead of typing them all out individually.

```
export interface OboSitesFormModel extends Pick<HttpEnrollee, 'oboSites' | 'enrolleeCareSettings' | 'enrolleeHealthAuthorities'> { }
```

Reply:
Yeah I was trying to fix this error that pops up when I used the `Pick`
```
Type '{ oboSites: any[]; }' is missing the following properties from type 'Pick<HttpEnrollee, "enrolleeCareSettings" | "enrolleeHealthAuthorities" | "oboSites">': enrolleeCareSettings, enrolleeHealthAuthoritiests(2739)
(property) oboSites: OboSite[]
```
It's located in the get json() method in the obo-sites-form-state.class.ts file

```  public get json(): OboSitesFormModel {
    if (!this.formInstance) {
      return;
    }

    const sites = this.formInstance.getRawValue();
    const oboSites = [
      sites.communityHealthSites,
      sites.communityPharmacySites,
      Object.keys(sites.healthAuthoritySites)
        .flatMap((healthAuthSiteCode: string) => sites.healthAuthoritySites[healthAuthSiteCode])
    ].flat();

    return { oboSites };
  }
  ```